ehitchhiking-ui/src/app/shared/components/menu/menu.component.sass

```diff
@@ -20,10 +17,8 @@
 .header
   display: flex
   justify-content: space-around
-  p
-    margin-top: 1.5rem
-    span
-      font: 100% $font-stack
+  span
+    font: 100% $main-font
```

Comment:
Pls set font to the html and body + rename $main_font to $app-font because we have no non main font

Reply:
done